index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Zilola's portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <header id="home-header">
+      <nav id="toolbar-container">
+        <h1 class="logo-placeholder">My Logo</h1>
+        <button class="burger"></button>
+      </nav>     
+    </header>
+    <main id="home-main">
+      <h2 class="headline-item" id="greeting">I’m Alexandra.<br /> Glad to see you!</h2>
+      <p class="headline-item" id="about-me">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>      
+      <section id="social-media">
+        <p class="headline-item" id="connect">LET'S CONNECT</p>
+        <div class="headline-item" id="icons">
+          <img src="./images/twitter.svg" href="https://twitter.com/ZilolaNazarova6" />
+          <img src="./images/Linkedin icon.svg" href="https://www.linkedin.com/in/zilola-nazarova-a64858265/" />
+          <img src="./images/medium.svg" href="#" />
+          <img src="./images/github.svg" href="https://github.com/Zilola-Nazarova" />
+          <img src="./images/Vector.svg" href="#" />
+        </div>       
```

Comment:
- [ ] Dear @Zilola-Nazarova, The list of icons better use a `ul` tag to group them, and `li` for each icon, then wrap each icon with a link (`a` tag).

```html
<ul class="headline-item" id="icons">
          <li><a><img src="./images/twitter.svg" href="https://twitter.com/ZilolaNazarova6" /></a></li>
                    <li><a><img src="./images/Linkedin icon.svg" href="https://www.linkedin.com/in/zilola-nazarova-a64858265/" /></a></li>
                    <li><a><img src="./images/medium.svg" href="#" /></a></li>
                    <li><a><img src="./images/github.svg" href="https://github.com/Zilola-Nazarova" /></a></li>
                   <li><a> <img src="./images/Vector.svg" href="#" />          </a></li>
        </ul>   

```